examples/external-data/tests/customerService.test.ts

```diff
@@ -131,14 +134,20 @@ describe("mock-customer-service", () => {
 
 	// We have omitted `@types/supertest` due to cross-package build issue.
 	// So for these tests we have to live with `any`.
-	it("register-for-webhook: Complete data flow", async () => {
+	// Skipping to close off the broadcast-signal loop. Tested manually and it works well.
+	// Unclear why localServiceApp is failing to post right for this test.
+	it.skip("register-for-webhook: Complete data flow", async () => {
```

Comment:
As discussed offline, jest's `fail()` function [seems to be broken](https://github.com/jestjs/jest/issues/11698). The types say it's there but it's actually not. Removing the catch block that calls it exposes the real errors happening in the tests. Let's fix those before merging the PR.